CDCgov/prime-simplereport
# Johanna/5659 banner up notification

# DEVOPS PULL REQUEST

## Related Issue

- Resolves #5741 

## Changes Proposed

- Create a custom action that sends an email to the support team as part of the maintenance workflow when the maintenance banner is either displayed or taken down.

## Additional Information

- The custom action will be using the SimpleReport SendGrid account as email sender. A new token with restricted permissions has been created for this purpose.

## Testing

- Add your email to the `NOTIFICATION_EMAIL_RECIPIENT_TEST` property. You will need Merethe's help to do so as only leads have access to modify properties.
- Run the maintenance workflow using this branch as base in any of the lower environments and. verify you get the email when the banner is either being put on or off.
<img width="932" alt="Screenshot 2023-06-20 at 9 28 50 AM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/6202d363-e6fb-4cb0-b3d7-bf211b815708">
<img width="630" alt="Screenshot 2023-06-20 at 9 28 25 AM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/b66d92b0-9f11-4584-b019-03afb8f6c3b6">
